src/Microsoft.VisualStudio.LanguageServices.Razor/DefaultTagHelperCompletionService.cs

```diff
@@ -194,8 +194,11 @@ public override ElementCompletionResult GetElementCompletions(ElementCompletionC
             {
                 foreach (var completionTagName in elementCompletions.Keys)
                 {
-                    if (completionTagName.StartsWith(prefix, StringComparison.OrdinalIgnoreCase))
+                    if (elementCompletions[completionTagName].Count > 0 || 
```

Comment:
Why do we need to check `count > 0`? I thought we were just saying catch alls wouldn't be offered as completions?

Also this is order dependent. It will only add a single one of the catch alls.